sync/src/relayer/compact_block_process.rs

```diff
@@ -64,17 +64,23 @@ impl<'a> CompactBlockProcess<'a> {
         let header = compact_block.header().into_view();
         let block_hash = header.hash();
 
-        // Only accept blocks with a height greater than tip - N
-        // where N is the current epoch length
         let snapshot: &Snapshot = &self.relayer.shared.snapshot();
         let tip = snapshot.tip_header().clone();
         let epoch_length = snapshot.epoch_ext().length();
         let lowest_number = tip.number().saturating_sub(epoch_length);
 
+        // Only accept blocks with a height greater than tip - N
+        // where N is the current epoch length
         if lowest_number > header.number() {
             return Err(Error::Ignored(Ignored::TooOldBlock).into());
         }
 
+        // If the CompactBlock height is greater than tip + 2.
+        // In high probability the node is syncing.
+        if header.number() > tip.number() + 2 {
+            return Err(Error::Ignored(Ignored::TooHighBlock).into());
+        }
```

Comment:
We should log `Error::Ignored` in `try_process()` too.

Reply:
updated